packages/web/src/app/app.component.ts

```diff
@@ -1,15 +1,19 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 import { TestService } from '@common';
-import { Observable, map } from 'rxjs';
+import type { Observable } from 'rxjs';
+import { map } from 'rxjs';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  title = 'web';
   protected readonly testModelNames$: Observable<string[]>;
+
+  @Input()
+  title = 'web';
```

Comment:
I suggest try to be consistent, if you want to write properties without class modifiers, so, pls, write all properties without it, or vice versa.